skimage/transform/hough_transform.py

```diff
@@ -258,14 +213,19 @@ def hough_line(img, theta=None):
     .. plot:: hough_tf.py
 
     """
+    if img.ndim != 2:
+        raise ValueError('The input image `img` must be 2D.')
+
     if theta is None:
         # These values are approximations of pi/2
         theta = np.linspace(-np.pi / 2, np.pi / 2, 180)
-    return _hough_line(img, theta)
+
+    return _hough_line(img, theta=theta)
 
 
-def probabilistic_hough_line(img, threshold=10, line_length=50,
-                             line_gap=10, theta=None):
+# Wrapper for Cython allows function signature introspection
```

Comment:
Remove also :). I mean, we have a large number of wrappers, no sense to me to highlight this for all of them.